src/store/tests/test-data/common/manoeuvres/manoeuvres.selectors.ts

```diff
@@ -1,6 +1,6 @@
-// import { get } from 'lodash';
-// import { ManoeuvreUnion } from '../../../../../shared/unions/test-schema-unions';
-//
-// export const getReverseLeftSelected = (manoeuvres: ManoeuvreUnion) => {
-//   return get(manoeuvres, 'reverseLeft.selected', false);
-// };
+import { get } from 'lodash';
+import { ManoeuvreUnion } from '../../../../../app/shared/unions/test-schema-unions';
```

Comment:
might be a good place to start introducing import paths in tsconfig as per https://decembersoft.com/posts/say-goodbye-to-relative-paths-in-typescript-imports/